ocaml/idl/datamodel_certificate.ml

```diff
@@ -64,8 +64,16 @@ let t =
       ; field ~qualifier:StaticRO ~lifecycle ~ty:DateTime "not_after"
           ~default_value:(Some (VDateTime Date.never))
           "Date before which the certificate is valid"
-      ; field ~qualifier:StaticRO ~lifecycle ~ty:String "fingerprint"
+      ; field ~qualifier:StaticRO
+          ~lifecycle:
+            [(Published, rel_stockholm, ""); (Deprecated, "24.17.0", "")]
```

Comment:
Tentatively, this should be:
```suggestion
      ; field ~qualifier:StaticRO
          ~lifecycle:
            [(Published, rel_stockholm, ""); (Deprecated, "24.18.0", "")]
```

Reply:
Ahh good point. What's the exact procedure for this, just bump the minor version (for new functionality)? Because won't the exact version depend on when it is released/what is added alongside it in the final build?

Reply:
Yes, this is manual and brittle currently :(